lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
Of course, you are the one who wants this merged in, not me! And of course my quality standards apply, not yours.

I'd say this is pretty standard in open-source, maybe not in the projects you're used to contributing I guess.